docs/source/models.rst

```diff
@@ -1774,22 +1792,24 @@ The following examples show how to create a deployment in ACI. Please, ensure yo
     import json
     # `sample_input` is a JSON-serialized pandas DataFrame with the `split` orientation
     sample_input = {
-        "columns": [
-            "alcohol",
-            "chlorides",
-            "citric acid",
-            "density",
-            "fixed acidity",
-            "free sulfur dioxide",
-            "pH",
-            "residual sugar",
-            "sulphates",
-            "total sulfur dioxide",
-            "volatile acidity"
-        ],
-        "data": [
-            [8.8, 0.045, 0.36, 1.001, 7, 45, 3, 20.7, 0.45, 170, 0.27]
-        ]
+        "input_data": {
```

Comment:
This is a breaking change, right? 
Can you provide a version context of when this change takes place (version number of the SDK) and provide a note on a lack of backwards compatibility (or, if backwards compatibility is handled, provide some indication that this is the case)?

Reply:
I think this functionality has been incorrectly documented here, cause this is the way it has worked from the beginning. The key `input_data` has always been required. I'm updating this documentation for that reason. I also added the note about this payload being different from the one MLflow uses. Do you think we should add something else?

Reply:
@BenWilson2  let me know what's your opinion around this.

Reply:
Thanks for clarifying / correcting the docs here!